src/components/faq/faq.tsx

```diff
@@ -1,18 +1,171 @@
-import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
-import { Box, Divider, Typography } from '@mui/material';
-import React from 'react';
+'use client';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import {
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Typography,
+} from '@mui/material';
+import MuiAccordion, { AccordionProps } from '@mui/material/Accordion';
+import { styled } from '@mui/material/styles';
+import { Lato } from 'next/font/google';
+import React, { useState } from 'react';
 import './faq.scss';
+const lato = Lato({
+  subsets: ['latin'],
+  weight: ['400', '700'],
+});
+
+// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Why disable Eslint error instead of fixing it?

Reply:
Couldn't find a better solution for this one, If I change to camelCase I won't be able to use it as a component

Reply:
Addressed in [1cee54a](https://github.com/OpenBCca/openbc-web/pull/92/commits/1cee54a8cfd56c4d1c89cb1ce9c8d391f0bca01e)